osu.Game/Screens/Menu/IntroCircles.cs

```diff
@@ -15,8 +22,8 @@ public class IntroCircles : IntroScreen
 
         protected override string BeatmapFile => "circles.osz";
 
-        private const double delay_step_one = 2300;
-        private const double delay_step_two = 600;
+        private const double load_menu_delay = 2400;
```

Comment:
can you explain the 100ms change here?

Reply:
Seems that the current load menu delay is a bit earlier with my new changes (start intro at sequence load completion), so I optimized the delay a bit, should be no harm?

Reply:
When variables change, I'd want to understand why, not just "seems to be different".

Reply:
@iiSaLMaN i hope you understand i *cannot* go further without this being resolved. are you planning on looking into this?

Reply:
yeah sorry for this delay. I didn't have time to look into this, will check it after finishing from other PR reviews atm.